tests/gc/cyclecollector.nim

```diff
@@ -16,6 +16,5 @@ proc main =
     var leaf = "this is the leaf. it allocates"
     let x = createCycle(leaf)
     let y = createCycle(leaf)
-  echo "done ", getOccupiedMem()
```

Comment:
Is this intended?

Reply:
`cyclecollector.nim` was never run before, only compiled. If you run it without an output check, test will fail since it does print something, so I removed the output altogether. `getOccupiedMem()` can vary by system so there's no guaranteed output to check against. Just printing done is pointless, return value check is sufficient.